tests/utils.py

```diff
@@ -970,6 +970,8 @@ def snapshot_context(token, ignores=None, tracer=None, async_mode=True, variants
         token = "{}_{}".format(token, variant_id) if variant_id else token
 
     ignores = ignores or []
+    # implementation of _dd.tracer_kr is flaky, so we ignore it by default
+    ignores.append("_dd.tracer_kr")
```

Comment:
you can ignore in the docker-compose.yml https://github.com/DataDog/dd-trace-py/blob/2.x/docker-compose.yml#L146